db/migrate/20200525073201_drop_puppet_class_id_from_look_up_key.rb

```diff
@@ -0,0 +1,5 @@
+class DropPuppetClassIdFromLookUpKey < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :lookup_keys, :puppetclass_id
```

Comment:
You need to add the column type for this to be reversible:
https://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method

Reply:
@tbrisker thanks! I hope it's ok now.